src/max_flow/maxFlow_process.cpp

```
@@ -0,0 +1,87 @@
+
+    std::ostringstream log;
+    std::ostringstream err;
+    std::ostringstream notice;
+
+    clock_t start_t = clock();
+    pgrouting::drivers::do_maxFlow(
+            edges_sql? edges_sql : "",
+            combinations_sql? combinations_sql : "",
+            starts, ends,
+
+            which,
```

Cppcheck (2.19.0)

[style] The function 'pgr_alloc' is never used.

(unusedFunction)